spec/unit/rack/notice_builder_spec.rb

```diff
@@ -69,5 +69,15 @@
       notice = notice_builder.build_notice(AirbrakeTestError.new)
       expect(notice[:environment]["SOME_KEY"]).to eq("SOME_VALUE")
     end
+
+    it "applies custom builders for notice" do
```

Comment:
`"runs builders against notices"` sounds more understandable IMO (or something similar). "Applies" is a bit confusing.